### PR TITLE
Fix AWSRecreateSG EC2 breaking change bug

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-33573)

## Description
AWSRecreateSG fails after the recent changes to `AWS - EC2` [(version 1.3.0)](https://github.com/demisto/content/pull/31416/)
The problems are caused by the change in the object returned when `AWS - EC2` commands are called using the `demisto.executeCommand` function in the script . This PR uses a new path to extract the required data correctly.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by johnnywilkes — APPROVED** [review]
tested in XSIAM and XSOAR.  This makes the script no longer backwards compatible, but I don't see this as a concern

**Review by JudahSchwartz — APPROVED** [review]

File: Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py [hunk 1]
```diff
@@ -86,13 +86,6 @@ def sg_fix(sg_info: list, port: int, protocol: str, assume_role: str, instance_t
                     and rule['IpProtocol'] == protocol
                 ):
                     change = True
-                elif (
```
Comment (johnnywilkes) [inline Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py:112]: this is not a duplicate, please don't remove

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
please re-add this block

File: Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py (hunk 1 above)
Comment (johnnywilkes) [reply Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py:112]: This elif is when the offending rule references the port exactly.  The next elif is when the port i part of a range the therefore we need to run it past the split_rule() function.

Comment (jlevypaloalto) [reply Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py:112]: The `elif` block I removed is a duplicate of the `if` block before it not the `elif` block after it.

Comment (johnnywilkes) [reply Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py:112]: you are right, good catch

**Review by johnnywilkes — APPROVED** [review]

**Pull request merged**